scripts/python/services/test_GithubService.py

```diff
@@ -10,58 +10,68 @@
 USER_ACCESS_REMOVED_ISSUE_TITLE = "User access removed, access is now via a team"
 
 
+@patch("gql.transport.aiohttp.AIOHTTPTransport.__new__", new=MagicMock)
```

Comment:
Had to add these two additional patches for all the tests to prevent live calls from being made.

Passing in the `new` keyword argument means that the mock isn't passed into the function as a parameter.

Really useful for all the current tests that require these mocks to exists, but don't make any use of them within the test 🥳